portfolio/src/main/webapp/script.js

```diff
@@ -34,3 +34,15 @@ async function showMeSomething(){
     const bingBong = document.getElementById('test-container');
     bingBong.innerText = textFromResponse;
 }
+
+async function generate(){
+    const responseFromServer = await fetch("/hello");
+    const arr = await responseFromServer.json();
+
+    const randomIndex = Math.floor(Math.random() * arr.length);
+
+    const randElement = document.getElementById("rand-element");
+    console.log(arr);
+    randElement.innerText = arr[randomIndex];
```

Comment:
Optional question, feel free to ignore. We can chat about it in 1-1:
what if you needed to display all elements from arr, not just one? What if each element should be represented as non-trivial HTML structure (a row with an icon, text and button inside)? What are possible ways to achieve this?